mlrun/datastore/datastore_profile.py

```diff
@@ -30,7 +30,7 @@
 class DatastoreProfile(pydantic.BaseModel):
     type: str
     name: str
-    _private_attributes: typing.List = ()
+    _private_attributes: list = ()
```

Comment:
Unrelated to this change but it's a wrong type hint (empty tuple assignment), pydantic handles it in an inconsistent way